content/shmem_team_get_config.tex

```diff
@@ -5,18 +5,21 @@
 \begin{apidefinition}
 
 \begin{Csynopsis}
-int @\FuncDecl{shmem\_team\_get\_config}@(shmem_team_t team, shmem_team_config_t *config);
+int @\FuncDecl{shmem\_team\_get\_config}@(shmem_team_t team, long config_mask, shmem_team_config_t *config);
 \end{Csynopsis}
 
 \begin{apiarguments}
   \apiargument{IN}{team}{An \openshmem team handle.}
+  \apiargument{IN}{config\_mask}{
```

Comment:
This should be a OUT

Reply:
Since the memory for config is allocated by user, it has to be in ?

Reply:
@naveen-rn -- The description for this mask reads like an input. Why do you think it should be `OUT`?

Reply:
Initially, I was under the assumption that the memory of the ```shmem_config_t``` mask is allocated by the implementation. But, after discussing with Pasha, it looks like the implementation should just update the values, and the allocation should be handled by the users. 
Since, this is the case, I think IN is fine,

Reply:
@BryantLam OpenSHMEM does not define release function for ```shmem_config_t```, as a result openshmem library should not be allocation it all together. User has to allocate memory for the struct and pass it to the library. So, the structure is always in.